package.json

```diff
@@ -4,18 +4,23 @@
   "version": "0.0.6",
   "description": "A suite of web components for a mutation testing report.",
   "main": "dist/mutation-test-elements.js",
+  "workspaces": [
+    "packages/*"
+  ],
+  "nx": {
```

Comment:
Why do we need config for `nx` in 2 places? Both `nx.json` and the `nx` key here.

Reply:
I've added a comment, but essentially this is to create a 'root' project. Because we have root tasks that need to run before submodule tasks can run (like schema:test depends on root:build)